manifests/a/AcerIncorporated/AcerProductRegistration/2.0.3030.0/AcerIncorporated.AcerProductRegistration.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: AcerIncorporated.AcerProductRegistration
+PackageVersion: 2.0.3030.0
+PackageLocale: en-US
+Publisher: Acer Incorporated
+PackageName: Acer Product Registration
+License: Copyright (c) Acer Incorporated

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate